packages/shorebird_cli/test/src/commands/release/release_android_command_test.dart

```diff
@@ -888,5 +920,71 @@ Note: ${lightCyan.wrap('shorebird patch android --flavor=$flavor --target=$targe
       expect(exitCode, equals(ExitCode.success.code));
       verifyNever(() => logger.confirm(any()));
     });
+
+    test('error when the app bundle cannot be found ', () async {
+      shorebirdAndroidArtifacts = MockShorebirdAndroidArtifacts();
+      when(
+        () => shorebirdAndroidArtifacts.findAppBundle(
+          project: any(named: 'project'),
+          flavor: any(named: 'flavor'),
+        ),
+      ).thenThrow(
+        ArtifactNotFoundException(
+          artifactName: 'app-release.aab',
+          buildDir: 'buildDir',
+        ),
+      );
+      final exitCode = await runWithOverrides(command.run);
+      verify(
+        () => logger.err('Artifact app-release.aab not found in buildDir'),
+      ).called(1);
+      expect(exitCode, ExitCode.software.code);
+    });
+
+    test('error when the apk cannot be found ', () async {
+      shorebirdAndroidArtifacts = MockShorebirdAndroidArtifacts();
+      when(
+        () => shorebirdAndroidArtifacts.findAppBundle(
+          project: any(named: 'project'),
+          flavor: any(named: 'flavor'),
+        ),
+      ).thenReturn(File('app-release.aab'));
```

Comment:
Is this override here because we try to find an aab before looking for an apk?

Reply:
Exactly, so the `findAppBundle` cannot fail, otherwise the test will not fail because of the reason that we want it to fail

Reply:
👍  I'd maybe leave a small comment so that's clear